### DISCUSSION TITLE
What is the default UI.ColorScheme? How can i set it back to it?

### DISCUSSION THREAD

**Discussion by leeenglestone** [StereoKit/StereoKit#404] [Q&A] [answered]
Hi,

Firstly, I have been attempting to change the colour of UI.Buttons by changing the UI.ColorScheme before each one.. I don't know if this is the right way to do it? I could only get a very slight colour change anyway..

Secondly, After I have coloured the buttons I want to revert the UI.ColorScheme back to the original/default one for the remaining controls.. otherwise it effects the colour of a UI.Window I instantiate as well. 

- Is this the right way to colour buttons?
- What is the default UI.ColorScheme?
- Is there an easy way of changing the UI.ColorScheme to the default colour scheme?

Thanks

-- Lee

**Comment by maluoi** [comment, marked as answer]
Okay, it looks like these functions managed to escape documentation 😅 But this latest release added 'UI.Get/SetThemeColor', these are a little granular, but sounds like what you may need in this case! There's also a UI.Push/PopTint, but that generally looks pretty heavy handed.

There's improvements to theming coming up as well, as this has been a spot where SK has previously lacked.

**Reply by leeenglestone** [reply]
Ok - I will see if I can UI.GetThemeColor when the app first runs and make a note of it  and will look at that other stuff as well. Thanks!

**Reply by leeenglestone** [reply]
hmm so this takes some parameters. 

            Color color = UI.GetThemeColor(UIColor colorType, Color `colorGamma);`

How do I get the current Theme? i.e. what Windows look like by default (that kind of brown color) @maluoi

**Reply by leeenglestone** [reply]
Actually, let me try the PushTint, PopTint first..

**Reply by maluoi** [reply]
UIColor.Primary is the window header color! There's only 5 colors that SK uses for the theme, and they get reused across the whole UI. If you check the intellisense, it should give a decent impression of which color is used where.

UI.GetThemeColor should only take the first parameter, and is currently ignoring the second 😬  That's a typo on my part! I guess that's what I get for not documenting things 😅

**Reply by leeenglestone** [reply]
PushTint and PopTint did the trick as they maintained the original theme afterwards
